docs/supported_platforms_and_configurations.md

```diff
@@ -37,7 +37,7 @@ For more information see [conan-io/conan-docker-tools](https://github.com/conan-
 ## Windows
 
 - Compilers: Visual Studio:
-  - 2015 (19.0.24215)
+  - 2015 (null)
```

Comment:
2015 should be removed from the list completely.

Reply:
Yes, but we can't do it here, it will require a new release for c3i